sensor_msgs/PointCloud2 input_cloud
---
robot_manipulation/GraspCandidates candidates
